Fixes #9255 - Explain now to handle empty strings and nulls (#9269)
MicrosoftDocs/PowerShell-Docs: reference/docs-conceptual/learn/deep-dives/everything-about-switch.md

```

For this example, the value of `$day` matches one of the numeric values, then the correct name is

> assigned to `$result`. we're only doing a variable assignment in this example, but any PowerShell

Reviewer: ```suggestion
assigned to `$result`. We're only doing a variable assignment in this example, but any PowerShell
```

}
```

> we're placing the value on the PowerShell pipeline and assigning it to the `$result`. You can do

Reviewer: ```suggestion
We're placing the value on the PowerShell pipeline and assigning it to the `$result`. You can do
```

### Regex $matches

> We need to revisit regex to touch on something that'sn't immediately obvious. The use of regex

Reviewer: ```suggestion
We need to revisit regex to touch on something that isn't immediately obvious. The use of regex
```

### Constant expression

Lee Dailey pointed out that we can use a constant `$true` expression to evaluate `[bool]` items.

> Imagine if we've several boolean checks that need to happen.

Reviewer: ```suggestion
Imagine if we have several boolean checks that need to happen.
```

### Hashtables

One of my most popular posts is the one I did on [hashtables][hashtables]. One of the use cases for

> a `hashtable` is to be a lookup table. that's an alternate approach to a common pattern that a

Reviewer: ```suggestion
a `hashtable` is to be a lookup table. That's an alternate approach to a common pattern that a
```
